selfdrive/thermald/fan_controller.py

```diff
@@ -93,10 +94,10 @@ def update(self, max_cpu_temp: float, ignition: bool) -> int:
       self.controller.reset()
 
     fan_pwr_out = -int(self.controller.update(
-                      setpoint=75,
-                      measurement=max_cpu_temp,
-                      feedforward=interp(max_cpu_temp, [60.0, 100.0], [0, -80])
-                    ))
+                                        setpoint=self.target,
+                                        measurement=max_cpu_temp,
+                                        feedforward=interp(self.target, [60, 100], [-80, 0])
```

Comment:
This interp is wrong. To make it match the old controller the result would need to be 30.

We thought about it a bit more, and changing it to an actual PID doesn't make it easier to reason about. It would imply that it's possible to change the setpoint without retuning the values, which is probably not true. If you do that you would need to find the relationship between kp and ff that works at all setpoints.

Since we don't plan on doing that anytime soon we'll just leave the controller as is.